### ISSUE TITLE
Address admin tests

### ISSUE THREAD

**Issue by shaun-technovation** [Iridescent-CM/technovation-app#3852]
The following tests should be updated/fixed, or removed:
```
Team submissions editable toggles team roster controls Mentor dashboard Toggle off
  # Temporarily disabled with xfeature
  # ./spec/features/admin/team_building_toggle_spec.rb:142

Team submissions editable toggles team roster controls Viewing team as student Toggle on
  # Temporarily disabled with xfeature
  # ./spec/features/admin/team_building_toggle_spec.rb:72

Team submissions editable toggles team roster controls Viewing team as student Toggle off
  # Temporarily disabled with xfeature
  # ./spec/features/admin/team_building_toggle_spec.rb:78

Team submissions editable toggles team roster controls Viewing team as mentor Toggle off
  # Temporarily disabled with xfeature
  # ./spec/features/admin/team_building_toggle_spec.rb:193

Team submissions editable toggles team roster controls Viewing team as mentor Toggle on
  # Temporarily disabled with xfeature
  # ./spec/features/admin/team_building_toggle_spec.rb:187

Team submissions editable toggles team roster controls Mentor my teams page Toggle on
  # Temporarily disabled with xfeature
  # ./spec/features/admin/team_building_toggle_spec.rb:157

Team submissions editable toggles team roster controls Mentor my teams page Toggle off
  # Temporarily disabled with xfeature
  # ./spec/features/admin/team_building_toggle_spec.rb:164

Team submissions editable toggles team roster controls Student nav bar, no team Toggle off
  # Temporarily disabled with xfeature
  # ./spec/features/admin/team_building_toggle_spec.rb:28

Team submissions editable toggles team roster controls Student nav bar, no team Toggle on
  # Temporarily disabled with xfeature
  # ./spec/features/admin/team_building_toggle_spec.rb:19

Team submissions editable toggles team roster controls Student team page, with team Toggle on
  # Temporarily disabled with xfeature
  # ./spec/features/admin/team_building_toggle_spec.rb:44

Team submissions editable toggles team roster controls Student team page, with team Toggle off
  # Temporarily disabled with xfeature
  # ./spec/features/admin/team_building_toggle_spec.rb:50

Team submissions editable toggles team roster controls Viewing current team as student Toggle on
  # Temporarily skipped with xcontext
  # ./spec/features/admin/team_building_toggle_spec.rb:100

Team submissions editable toggles team roster controls Viewing current team as student Toggle off
  # Temporarily skipped with xcontext
  # ./spec/features/admin/team_building_toggle_spec.rb:107

Team submissions editable toggles team roster controls Viewing current team as mentor Toggle on
  # Temporarily skipped with xcontext
  # ./spec/features/admin/team_building_toggle_spec.rb:216

Team submissions editable toggles team roster controls Viewing current team as mentor Toggle off
  # Temporarily skipped with xcontext
  # ./spec/features/admin/team_building_toggle_spec.rb:222

Team submissions editable toggles team roster controls Viewing mentor profile as student Toggle off
  # Temporarily disabled with xfeature
  # ./spec/features/admin/team_building_toggle_spec.rb:129

Team submissions editable toggles team roster controls Viewing mentor profile as student Toggle on
  # Temporarily disabled with xfeature
  # ./spec/features/admin/team_building_toggle_spec.rb:123

Admin / chapter ambassador logging in as a user as a chapter ambassador Chapter ambassador logging in as a mentor
  # Blocked for chapter ambassadors
  # ./spec/features/admin/login_as_spec.rb:28

Admin / chapter ambassador logging in as a user as a chapter ambassador Chapter ambassador logging in as a student
  # Blocked for chapter ambassadors
  # ./spec/features/admin/login_as_spec.rb:10

Admin / chapter ambassador logging in as a user as a chapter ambassador Chapter ambassador logging in as a judge
  # Blocked for chapter ambassadors
  # ./spec/features/admin/login_as_spec.rb:46
```

**Comment by shaun-technovation** [comment]
There are three specs here that I forgot to address when I worked on this issue. The three specs are in: `spec/features/admin/login_as_spec.rb`.

I was looking at our outstanding pending specs (there are currently 39), and wanted to fix some of them, I found these three, but couldn't find the issue related to them right away, then I realized this issue was closed already! Anywho, these three overlooked specs are being addressed in PR #4076.
